Source/driver/Castro_advance_sdc.cpp

```diff
@@ -95,7 +95,6 @@ Castro::do_advance_sdc (Real time,
 #endif
 
       if (apply_sources()) {
-#ifndef AMREX_USE_GPU
```

Comment:
Do you know why this was ifdef'd out?

Reply:
well, it doesn't work with GPUs, so we just didn't want to compile it, but this entire block is already wrapped in an `#ifndef AMREX_USE_GPU`, so clang-tidy said this was a dupe.